src/util.hpp

```diff
@@ -53,6 +53,7 @@ extern bool askfor_aux(std::string *buf, std::size_t max_len);
 extern bool_ askfor_aux(char *buf, int len);
 extern bool_ askfor_aux_with_completion(char *buf, int len);
 extern bool_ get_string(cptr prompt, char *buf, int len);
+extern bool_ get_filename(cptr prompt, char *buf, int len);
```

Comment:
Here, the "extern" should just be left off. It's only laziness that's prevented me from doing that for every single (non-extern-C) function already.